storage/filestore/dir.go

```diff
@@ -40,18 +40,18 @@ func NewDir(path string) (*Dir, error) {
 	}
 
 	return dir, errs.Combine(
-		os.MkdirAll(dir.blobdir(), dirPermission),
+		os.MkdirAll(dir.blobsdir(), dirPermission),
 		os.MkdirAll(dir.tempdir(), dirPermission),
-		os.MkdirAll(dir.trashdir(), dirPermission),
+		os.MkdirAll(dir.garbagedir(), dirPermission),
 	)
 }
 
 // Path returns the directory path
 func (dir *Dir) Path() string { return dir.path }
 
-func (dir *Dir) blobdir() string  { return filepath.Join(dir.path, "blob") }
-func (dir *Dir) tempdir() string  { return filepath.Join(dir.path, "tmp") }
-func (dir *Dir) trashdir() string { return filepath.Join(dir.path, "trash") }
+func (dir *Dir) blobsdir() string   { return filepath.Join(dir.path, "blobs") }
+func (dir *Dir) tempdir() string    { return filepath.Join(dir.path, "temp") }
+func (dir *Dir) garbagedir() string { return filepath.Join(dir.path, "garbage") }
```

Comment:
Why the rename?

Reply:
Jeff and JT said, that we should try to take other paths to ensure we are not writing in the "old" directories.
More of a overeager sanity check.

Reply:
In this way we don't have to worry about losing track of migrations, running them again, and deleting all of the live data.